src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler/Configuration.php

```diff
@@ -132,9 +143,9 @@ public function isInRegexMode()
     /**
      * @return bool
      */
-    public function verboseOutput()
+    public function verboseOutput($group)
```

Comment:
bc break?

Reply:
Yes, but the class is marked as internal